CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.5] - 2024-03-04
+### Changed
```

Comment:
We can say `Added` as there is no functionality change.

Reply:
Updated